config/database.php

```diff
@@ -50,7 +50,7 @@
             'charset' => 'utf8mb4',
             'collation' => 'utf8mb4_unicode_ci',
             'prefix' => '',
-            'strict' => true,
+            'strict' => false,
```

Comment:
This is not necessary for what you are doing. Please revert this back. I will add a comment on your queries to fix the issue you where having.

Reply:
Reverted in ae4298f